src/scrappers/source/proCyclingStats/raceStages.js

```
@@ -20,109 +19,165 @@ import { config } from "@scrappers/html/config-puppeteer";
+        case "arrival":
+          details[key] = value;
+          break;
+        case "distance":
+        case "verticalMeters":
+          details[key] = Number(value) || null;
+          break;
+        case "date":
+          details[key] = formatDate(year, value, "/");
+          break;
+        case "parcoursType":
+          const span = td.querySelector("span");
```

ESLint

[error] Unexpected lexical declaration in case block.

(no-case-declarations)